authui/src/authflowv2/components/otp-input.css

```diff
@@ -0,0 +1,124 @@
+@layer components {
+  :root {
+    --otp-input__spacing: 21px;
+
+    --otp-input-text-digit__ring-color: var(--color-grey-200);
```

Comment:
`--otp-input` is the BEM block. `text-digit-ring-color` is the BEM element. So it should be `--otp-input__text-digit-ring-color`.